rosplane_sim/params/fixedwing.yaml

```diff
@@ -98,7 +98,7 @@ delta_t0: 0.4
 wind_speed_topic: "gazebo/wind_speed"
 truthTopic: "truth"
 
-# Forces and Moments
+# Forces anI don't think that I understand the limitations of this XFLR model well enough.d Moments
```

Comment:
```suggestion
# Forces and Moments (I don't think that I understand the limitations of this XFLR model well enough.)
```

Reply:
I didn't even mean to change this file. I've reverted it back.